### PR TITLE
county level scraper for OK

### PR DESCRIPTION
**Pull request by paulboal**
This is a county-level scraper for OK from the public health website that was already documented: https://www.ok.gov/health/Prevention_and_Preparedness/Acute_Disease_Service/Disease_Information/Coronavirus_Disease_2019/Oklahoma_Response_to_Coronavirus_Disease_2019/index.html

### CODE REVIEW

File: scrapers/USA/OK/index.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+          counties[countyName] += casesState;
+        } else {
+          counties[countyName] = casesState;
+        }
+      }
+    });
+
+    const countiesList = [];
+
+    for (const c in counties) {
```
Comment (lazd) [inline scrapers/USA/OK/index.js]: We have `transform.objectToArray()` that does this for ya

Comment (paulboal) [reply scrapers/USA/OK/index.js]: Got it.

**Comment by lazd** [comment]
I made 2 tiny changes and renamed the functions I just refactored, and this is going in! Thank you!!

**Comment by lazd** [comment]
In my haste of merging your excellent contribution, I forgot to ask that you add yourself as the maintainer if you'd like to be credited on the site and are willing to maintain it (like done here): https://github.com/lazd/coronadatascraper/blob/master/scrapers/BRA/index.js#L15-L24

Additionally, we've also added a `source` field where we put the `name`, `url` or anything else that's relevant so we can cite the source properly. I see that, for this PR, your URL is the same as your source (for others, the source may be, say, the Department of Health, but the URL is ArcGIS's server). In this case, I would add:

```
source:  {
  name: 'Oklahoma State Department of Health'
}
```

**Pull request merged**